hw01-maven/src/main/java/HelloOtus.java

```diff
@@ -0,0 +1,9 @@
+import com.google.common.base.Strings;
+
+public class HelloOtus {
```

Comment:
Класс стоит поместить в пакет. 
squid:S1220
Unnamed packages are provided by the Java platform principally for convenience when developing small or temporary applications or when just beginning development.
To enforce this best practice, classes located in default package can no longer be accessed from named ones since Java 1.4.